staging/src/k8s.io/apimachinery/pkg/util/clock/clock.go

```diff
@@ -21,11 +21,18 @@ import (
 	"time"
 )
 
+// PassiveClock allows for injecting fake or real clocks into code
```

Comment:
I recommend leaving the clock work to a separate PR.  We already have a separate PR for refactoring `k8s.io/apimachinery/pkg/util/clock` to reveal PassiveClock: #80955.